lib/publiccloud/k8s_provider.pm

```diff
@@ -0,0 +1,78 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Helper class for amazon connection and authentication
+#
+# Maintainer: Clemens Famulla-Conrad <cfamullaconrad@suse.de>, qa-c team <qa-c@suse.de>
+
+package publiccloud::k8s_provider;
+use Mojo::Base -base;
+use testapi;
+use mmapi 'get_current_job_id';
+
+has key_id => undef;
```

Comment:
this line and two below already in `provider_client` , right ?

Reply:
What I found is that in our k8s tests we don't are provided by key_id so the code always call to the vault to create them. So I removed this from k8s_provider.

Reply:
Done